### ISSUE TITLE
HTML vellip; not displayed

### ISSUE THREAD

**Issue by ferrall** [wkhtmltopdf/wkhtmltopdf#4356]
**wkhtmltopdf version(s) affected**:  0.12

**OS information**
Windows

**Description**
The HTML character code for vertical ellipsis is not rendered.  
The code is &amp;vellip; and is displayed on the next line:
&vellip;
In the PDF it is displayed as:
􀀀
I have tried the hex and decimal versions.  Some other HTML codes are rendered properly.  So I'm not sure how many character codes this applies to.

**How to reproduce**
Simply put the HTML code above in your document

**Comment by PhilterPaper** [comment]
Do you know what font you're using? I wouldn't be surprised if it was missing from several fonts, as it's really more of a mathematics symbol than something used in general text. You might try explicitly naming the font face you use.

**Comment by ferrall** [comment]
Thanks.

I did some experimenting.  It is not a font issue.  I removed all CSS styles completely and it would not render.  

But I commented out the MathJax script in the HTML file and vellip worked fine.  But if I produce the PDF in Acrobat with MathJax it renders properly (or Save as PDF works too).  I'm trying wkhtmltopdf because I have other problems using Acrobat to produce PDF with MathJax, so it's an annoying but minor problem.

The script itself does not cause the problem (based on a simple example), so there is some interaction causing the problem.

**Comment by PhilterPaper** [comment]
CSS has nothing to do with it. If commenting out MathJax makes vellip appear in wkHTMLtoPDF, it sounds like MathJax is changing the font, to one that does not include vellip.

**Comment by ferrall** [comment]
MathJax is playing a role, but the issue only occurs when processing the html file through wkHTMLtoPDF.  vellip is displayed in HTML with MathJax.  It also displays in the PDF using Acrobat or Save-to-PDF in the browser.    So there seems to be an interaction between them.

On 11/05/2019 11:41 AM, Phil Perry wrote:

CSS has nothing to do with it. If commenting out MathJax makes vellip appear in wkHTMLtoPDF, it sounds like MathJax is changing the font, to one that does not include vellip.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fwkhtmltopdf%2Fwkhtmltopdf%2Fissues%2F4356%23issuecomment-491521566&data=02%7C01%7Cferrallc%40queensu.ca%7C3fca80ec851e41ca782b08d6d6271ee4%7Cd61ecb3b38b142d582c4efb2838b925c%7C1%7C0%7C636931860848493006&sdata=mlLjHiQN2RzFODrfOCnz3aVgR62fBcYmCjs%2F%2BHN%2BVAI%3D&reserved=0>, or mute the thread<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAB2OBJCR67TS757PK4JBXOTPU3SKFANCNFSM4HMFE6NQ&data=02%7C01%7Cferrallc%40queensu.ca%7C3fca80ec851e41ca782b08d6d6271ee4%7Cd61ecb3b38b142d582c4efb2838b925c%7C1%7C0%7C636931860848493006&sdata=kq8IZQqLE6TMc%2Bt0U5UTZhn4UPUp4ZRpqf0vj91sfd0%3D&reserved=0>.

--

==========================================
Christopher Ferrall
Professor of Economics
Queen's University
613-533-6658
econ.queensu.ca/pub/faculty/ferrall
==========================================
